newIDE/app/src/UI/MarkdownText.js

```diff
@@ -61,6 +61,7 @@ export const MarkdownText = (props: Props) => {
             'standalone-text-container': props.isStandaloneText,
           })}
           components={markdownCustomComponents}
+          linkTarget="_blank"
```

Comment:
interesting, the `makeMarkdownCustomComponents` should already apply a target blank if this is an `a`.
Is this not working?

Reply:
Discussed together: I didn't have installed the version you specified